doc/nrf/known_issues.rst

```diff
@@ -816,6 +816,13 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-5-0
+
+DRGN-15382: The SoftDevice Controller cannot be qualified on nRF52832
+  The SoftDevice Controller cannot be qualified on nRF52832.
+
+  **Workaround:** Upgrade to v1.5.1 (once available) or use the master branch.
```

Comment:
Do we need to specify a workaround here? Any other release than 1.5.0 should also work. @ru-fu  may have some input

Reply:
Yes, we should always try to give a workaround. Just telling people that something does not work is not user-friendly.

Reply:
Wait for the fix is not a workaround though.

Reply:
This page is an orphan anyway, and links go to the page on master.
And correct, waiting is not a workaround, but it's better to add it like this now and have it correct once 1.5.1 is out than to not add it and forget to update it when 1.5.1 is released ...